### PR TITLE
Add git diff example to Integration docs

### PR DESCRIPTION
**Pull request by JesseVermeulen123**
Part of https://github.com/sharkdp/bat/issues/448

Thanks to https://github.com/sharkdp/bat/pull/940

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,7 +131,13 @@ highlighting:
 git show v0.6.0:src/main.rs | bat -l rs
 ```
 
-Note that syntax highlighting within diffs is currently not supported. If you are looking for this, check out [`delta`](https://github.com/dandavison/delta).
```
Comment (SuperSandro2000) [inline README.md:134]: I really would love to keep the delta recommendation.

Comment (JesseVermeulen123) [reply README.md:134]: Sounds good. I considered keeping it.

File: README.md [hunk 2]
```diff
@@ -131,7 +131,15 @@ highlighting:
...
+#### `git diff`
+
+You can combine `bat` with `git diff` to view lines around code changes with proper syntax
+highlighting:
+```bash
+git diff --name-only --diff-filter=d | xargs bat --diff
+```
```
Comment (sharkdp) [inline README.md:140]: @eth-p Probably also a good place to refer to `batdiff` from `bat-extras`, right?

**Comment by sharkdp** [comment]
Thank you very much for your contribution!

File: README.md (hunk 2 above)
Comment (sharkdp) [reply README.md:140]: But we can already merge this as-is.

Comment (eth-p) [reply README.md:140]: This would be a good place to refer to it, yeah.

**Pull request merged**